dockerfiles/che/entrypoint.sh

```diff
@@ -291,27 +291,39 @@ init() {
 }
 
 add_cert_to_truststore() {
-  if [ "${CHE_SELF__SIGNED__CERT}" != "" ]; then
-    DEFAULT_JAVA_TRUST_STORE=$JAVA_HOME/lib/security/cacerts
-    DEFAULT_JAVA_TRUST_STOREPASS="changeit"
+  DEFAULT_JAVA_TRUST_STORE=$JAVA_HOME/lib/security/cacerts
+  DEFAULT_JAVA_TRUST_STOREPASS="changeit"
 
-    JAVA_TRUST_STORE=/home/user/cacerts
-    SELF_SIGNED_CERT=/home/user/self-signed.crt
+  JAVA_TRUST_STORE=/home/user/cacerts
+  SELF_SIGNED_CERT=/home/user/self-signed.crt
 
-    echo "Found a custom cert. Adding it to java trust store based on $DEFAULT_JAVA_TRUST_STORE"
-    cp $DEFAULT_JAVA_TRUST_STORE $JAVA_TRUST_STORE
+  echo "Found a custom cert. Adding it to java trust store based on $DEFAULT_JAVA_TRUST_STORE"
```

Comment:
I think the message is relevant for both cases. The difference is only that the `$JAVA_TRUST_STORE` will not be created again, as it was copied from `$DEFAULT_JAVA_TRUST_STORE` earlier. It is the same file in both cases.

Reply:
Well, I agree that it more or less relevant but in such case we'll see two the same entries, like:

```
Found a custom cert. Adding it to java trust store based on /etc/java/default-trust-store
Found a custom cert. Adding it to java trust store based on /etc/java/default-trust-store
```

instead it would be handier to see the actual operation like

```
Found a custom cert. Adding it to java trust store /home/che/trust-store based on /etc/java/default-trust-store
Found a custom cert. Adding it to java trust store /home/che/trust-store
```

It would help to understand and debug issues easier. WDYT?

Reply:
Fixed